src/composables/useFlawAffectsModel.ts

```diff
@@ -269,3 +289,7 @@ function areAffectsEqual(affect: ZodAffectType, otherAffect?: ZodAffectType, sho
   const excludeFields = pickBy((_, key) => !fieldsToExclude.includes(key));
   return equals(excludeFields(affect), excludeFields(otherAffect));
 }
+
+function areAffectsCvssEqual(affectCvss: ZodAffectCVSSType, otherAffectCvss: ZodAffectCVSSType) {
+  return equals(affectCvss, otherAffectCvss);
+}
```

Comment:
Is this abstraction providing any advantage than just using `equals` where needed?